src/components/ContentRadioList/ContentRadioList.stories.js

```diff
@@ -1,149 +1,151 @@
       <hr>
       <StorybookMobileDeviceSimulator :device="device">
-        <ContentRadioList v-model="selectedValue"
-                          id="exampleContentRadioList"
-                          :title="title"
-                          :options="options"
-                          :disabled="disabled"
-                          @input="onInput"
-        />
-        <hr>
-        <div style="margin: 20px">
-          Bound value: {{ selectedValue }}
-        </div>
+        <story />
       </StorybookMobileDeviceSimulator>
     </div>
   `,
 });
+deviceDecorator.argTypes = {
+  device: { control: { type: 'select', options: [...availableDevices] } },
+};
 
-const withCustomCheckedUncheckedIcons = () => ({
-  components: {
-    ContentRadioList,
-    CheckBoldIcon,
-    StorybookMobileDeviceSimulator,
+const ContentRadioListStories = {
+  component: ContentRadioList,
+  title: 'Components/ContentRadioList',
+  decorators: [withKnobs, deviceDecorator],
+  args: {
+    id: 'exampleContentRadioList',
+    dataTestId: 'content-radio-list',
   },
-  props: {
-    device: {
-      default: select('Simulated Mobile Device', [...availableDevices], availableDevices[0]),
-    },
-    title: {
-      default: text('Title', 'Example Title'),
+  argTypes: {
+    options: { control: 'object' },
+    value: { control: 'text' },
+    id: { control: 'text' },
+    dataTestId: { control: 'text' },
+    checkedIcon: {
+      control: {
+        type: 'text',
+      },
+      table: {
+        type: {
+          summary: null,
+        },
+      },
     },
-    disabled: {
-      default: boolean('Is Disabled?', false),
+    uncheckedIcon: {
+      control: {
+        type: 'text',
+      },
+      table: {
+        type: {
+          summary: null,
+        },
+      },
     },
   },
+};
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: {
+    ContentRadioList,
+    ...AllIcons,
+    RenderString,
+  },
   data() {
     return {
-      selectedValue: 'option1',
-      options: [
-        {
-          title: 'Option 1',
-          metaText: 'Description 1',
-          value: 'option1',
-        },
-        {
-          title: 'Option 2',
-          metaText: 'Description 2',
-          value: 'option2',
-        },
-        {
-          title: 'Option 3',
-          metaText: 'Description 3',
-          value: 'option3',
-        },
-      ],
+      selectedValue: args.value,
     };
   },
   methods: {
     onInput: action('Changed!'),
   },
+  watch: {
+    value() {
+      this.selectedValue = this.value;
+    },
+  },
   template: `
-    <div style="margin: 10px 50px 10px 50px;">
-      <h2><strong>ContentRadioList:</strong>&nbsp;A control that allows a user to select an option by showing all available options as a list of radio buttons.</h2>
+    <div>
+      <ContentRadioList v-model="selectedValue"
+                        :id="id"
+                        :dataTestId="dataTestId"
```

Comment:
This should be :data-test-id="dataTestId"

Reply:
changed